### PR TITLE
#151 prismic slices

### PR DESCRIPTION
**Pull request by defudef**
## Important
Please review it after #155 will be merged

### CODE REVIEW

File: packages/prismic/components/PrismicSlice.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  }
+
+  if (block.link_type !== undefined) {
+    return block.kind === 'image' ? 'image' : 'link'
+  }
+
+  return undefined
+}
+
+const transformBlock = (block) => ({
```
Comment (EugeneHlushko) [inline packages/prismic/components/PrismicSlice.vue]: Looks like `const transformedBlock = transformRepeatableBlock(block)` could save a few calls here

File: packages/prismic/rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import pkg from './package.json'
+import typescript from 'rollup-plugin-typescript2'
+import { terser } from 'rollup-plugin-terser'
+
+export function generateBaseConfig (pkg) {
+  return {
+    input: 'src/index.ts',
+    output: [
+      {
+      file: pkg.main,
```
Comment (EugeneHlushko) [inline packages/prismic/rollup.config.js]: looks like formatting is off in this file

**Review by EugeneHlushko — APPROVED** [review]
Looks good, left two minor comments hope they help

File: packages/prismic/components/PrismicSlice.vue (hunk 1 above)
Comment (defudef) [reply packages/prismic/components/PrismicSlice.vue]: Oh, that's an old file and it should be removed at all

File: packages/prismic/src/helpers/index.ts [hunk 3]
```diff
@@ -47,8 +47,25 @@ export const getBlocks = ({ data }: PrismicPage, blockName?: string): string | s
   return blockKeys.map(key => transformBlock(data[key]))
 }
 
-// TODO: slices
-export const getSlices = () => {}
+export const getSlices = ({ data }: PrismicPage, sliceType?: string) => {
```
Comment (andrzejewsky) [inline packages/prismic/src/helpers/index.ts]: missing return type

Comment (andrzejewsky) [reply packages/prismic/src/helpers/index.ts]: PrismicSlice ?

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**